src/openfl/text/TextField.hx

```diff
@@ -3055,6 +3055,14 @@ class TextField extends InteractiveObject implements IShaderDrawable {
 	
 	private function window_onTextInput (value:String):Void {
 		
+		if (!multiline) {
+			
+			// strip newlines when pasting the multi-line text,
```

Comment:
`...when pasting the multi-line text...` but the if check is `if (!multiline) {`

Reply:
well, yeah. when pasting multi-line _text_ in a non-multiline TextField